tests/lib/rules/jsx-no-literals.js

```diff
@@ -218,6 +218,20 @@ ruleTester.run('jsx-no-literals', rule, {
       `,
       options: [{noStrings: true, allowedStrings: ['&nbsp;']}]
     },
+    {
+      code: `
+        class Comp1 extends Component {
+          render() {
+            return (
+              <div>
+                &nbsp;
+              </div>
+            );
+          }
+        }
+      `,
+      options: [{noStrings: true, allowedStrings: ['&nbsp;']}]
```

Comment:
could we add a few more tests, one with leading and trailing spaces on `allowedStrings`, and a different one with them on the rendered text?

Reply:
apologies, i'm not sure how to action this request. could you say it another way?

for context, we happen to have a pre-existing test with spaces in `allowedStrings` that would fail if `allowedStrings` were not trimmed. here are all the combinations of spaces and not spaces with test coverage:

| tested | spaces in `allowedStrings` | spaces in rendered text |
| --- | --- | ---
| ✅  | ✅ | ✅  |
| ✅  | 🚫 | ✅ |
| ✅  | 🚫 | 🚫 |
| 🚫 | ✅ | 🚫 |

is that the test gap you'd like me to fix?

Reply:
Yes, exactly right :-)

Reply:
I'd also like a test with noStrings false, and an allowedStrings value :-)

Reply:
✅ I've made `noStrings: false` explicit in the existing test case